frameworks/PHP/phalcon/phalcon-micro.dockerfile

```diff
@@ -18,7 +18,7 @@ ENV PATH /composer:${PATH}
 ADD ./ /phalcon
 WORKDIR /phalcon
 
-RUN apt install -yqq php7.2-phalcon php7.2-dev
+RUN apt install -yqq wget php7.2-phalcon php7.2-dev
```

Comment:
Here it's installing php7.2-phalcon.
And the next lines download and build again phalcon